src/sass/app_2.scss

```diff
@@ -8,4 +8,6 @@
 @import 'app_2/_common/buttons.scss';
 @import 'app_2/_common/input_field.scss';
 @import 'app_2/_common/dropdown.scss';
+@import 'app_2/_common/contracts_popup.scss';
```

Comment:
Do we need to call `contracts_popup.scss` for this component?

Reply:
nope it doesn't depend on it. I thought it got here from the merge conflict, but I don't see any merges in commit history. That's weird. 😅

Reply:
I guess it should be removed then. :)